packages/backend/src/controllers/projectController.ts

```diff
@@ -306,7 +307,11 @@ export class ProjectController extends BaseController {
     /**
      * List group access for projects
      */
-    @Middlewares([allowApiKeyAuthentication, isAuthenticated])
+    @Middlewares([
+        authenticateServiceAccount,
+        allowApiKeyAuthentication,
```

Comment:
Should we perform a ` try {authenticateServiceAccount} catch{}`  inside allowApiKeyAuthentication ? 

This will save us duplicating this middlewire line on every method in controllers. Assuming we're going to support the same endpoints.